gradle.properties

```diff
@@ -2,7 +2,7 @@ org.gradle.caching=true
 puppycrawlCheckstyleVersion=8.18
 group=org.ballerinalang
 version=1.0.6-SNAPSHOT
-ballerinaLangVersion=2.0.0-alpha3-SNAPSHOT
+ballerinaLangVersion=2.0.0-alpha4-SNAPSHOT
 ballerinaTomlParserVersion=1.2.3-SNAPSHOT
 stdlibIoVersion=0.6.0-alpha2-SNAPSHOT
 stdlibOsVersion=0.8.0-alpha2-SNAPSHOT
```

Comment:
```suggestion
stdlibIoVersion=0.6.0-alpha3-SNAPSHOT
stdlibOsVersion=0.8.0-alpha3-SNAPSHOT
```

Reply:
Shall we update the rest of the dependencies as well?